libraries/tests/AWS.Lambda.Powertools.Metrics.Tests/ClearDimensionsTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.IO;
+using AWS.Lambda.Powertools.Common;
+using Moq;
+using Xunit;
+
+namespace AWS.Lambda.Powertools.Metrics.Tests;
+
+[Collection("Sequential")]
+public class ClearDimensionsTests
+{
+    [Fact]
+    public void WhenClearAllDimensions_NoDimensionsInOutput()
+    {
+        // Arrange
+        var methodName = Guid.NewGuid().ToString();
+        var consoleOut = new StringWriter();
+        Console.SetOut(consoleOut);
+
+        var configurations = new Mock<IPowertoolsConfigurations>();
+
+        var logger = new Metrics(
```

Comment:
Something we need to change later in all Metrics tests, the variable name should be metrics and not logger.